### DISCUSSION TITLE
Upload to git releases as apk

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [robertodoering/harpy#691] [Ideas]
It would be awesome if the app gets uploaded to fdroid or release section(as apk) cuz some peeps dont use google play & it is pain to download it from other sites

**Comment by robertodoering** [comment]
Thanks to help from @nplettenberg, apks are now built and uploaded on every release automatically :)
https://github.com/robertodoering/harpy/releases/

**Reply by HikaruIchijyo** [reply]
Signature d/f from playstore?
Cuz it is not updating

**Reply by robertodoering** [reply]
You won't be able to update the playstore releases but you'll be able to update manually installed releases

**Reply by HikaruIchijyo** [reply]
Got it

**Reply by HikaruIchijyo** [reply]
> Thanks to help from @nplettenberg, apks are now built and uploaded on every release automatically :)
> https://github.com/robertodoering/harpy/releases/

1 suggestion how about a build-in app updater? Especially for github?

**Comment by HikaruIchijyo** [comment]
> Thanks to help from @nplettenberg, apks are now built and uploaded on every release automatically :)
> https://github.com/robertodoering/harpy/releases/

Thanks it is helpful now
